File: app/src/main/java/com/bing/example/main/adapter/MainFragmentPagerAdapter.java
package com.bing.example.main.adapter;

import android.view.ViewGroup;

import java.util.ArrayList;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentPagerAdapter;

public class MainFragmentPagerAdapter extends FragmentPagerAdapter {
	private ArrayList<Fragment> mFragments;

	public MainFragmentPagerAdapter(FragmentManager fm, ArrayList<Fragment> fragments) {
		super(fm);
		mFragments = fragments;
	}

	@Override
	public Fragment getItem(int position) {
		return mFragments.get(position);
	}

	@Override
	public int getCount() {
		return mFragments.size();
	}

	@NonNull
	@Override
	public Object instantiateItem(@NonNull ViewGroup container, int position) {
		return super.instantiateItem(container, position);
	}
}

File: app/src/main/java/com/bing/example/main/home/MainFragmentPagerAdapter.kt
package com.bing.example.main.home

import android.view.ViewGroup

import java.util.ArrayList
import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter

class MainFragmentPagerAdapter(fm: FragmentManager, private val mFragments: ArrayList<Fragment>) : FragmentPagerAdapter(fm) {

        override fun getItem(position: Int): Fragment {
                return mFragments[position]
        }

        override fun getCount(): Int {
                return mFragments.size
        }

        override fun instantiateItem(container: ViewGroup, position: Int): Any {
                return super.instantiateItem(container, position)
        }
}
